plugins/api-docs-backend/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@backstage/plugin-api-docs-backend",
```

Comment:
To be honest I have a bit of a hard time placing this. I'm thinking it should maybe be `@backstage/plugin-search-backend-module-api-docs` instead? When I saw the PR I actually expected a backend plugin for storing definitions or something like that, this is more of a search addon imo.

Any thoughts on this @backstage/techdocs-core?

Reply:
What would happen if say someone needed to add backend support outside of search for `api-docs` in the future. Would they then merge the dedicated search module back into the backend? It seems pretty common plugins to evolve this way it seems.

Reply:
It would be completely fine for the search module to live separately from the plugin too. One of the pieces I'm thinking about here is that it's unfortunate for the search plugin deployment to have all these other plugins be part of its installed dependencies, even if you split the search plugin out to its own isolated backend. We don't really have a perfect naming pattern for this though, but I agree that `plugin-search-backend-module-api-docs` is pretty awkward and inverted.

Wonder if something like `plugin-api-docs-module-for-search` or some other way of flipping it around makes sense then? I agree it's nicer to keep this "close" to `api-docs`.

Reply:
Yeah thought process here was just picking up on the pattern on `plugin` and `plugin-backend` as a way to provide a place where other backend api-docs functionality could go in the future- but valid point on dependencies. Happy to rename to `plugin-api-docs-module-for-search` or `plugin-api-docs-search-module`. Is that the consensus here?

Reply:
@Rugvip Renamed this to `plugin-api-docs-module-search` let me know if that sounds good.

Reply:
Hmm, not sure, it feels confusing as a general pattern since it implies that it's a module that adds search capabilities to the `api-docs` plugin. Flagged it for discussion with @backstage/maintainers so will get back to you in a bit.

Reply:
Sounds good- I'll hold tight. Thanks!

Reply:
Discussed this a round with @backstage/maintainers and we're thinking it's best to have this be `plugin-search-backend-module-api-docs`. It really is a module for the search backend, and the fact that it happens to bring in data from one of our own plugins is secondary. Wanna double check with @backstage/techdocs-core that you think this is alright as well, and whether you think there should be an additional affix or anything in the name, for example `plugin-search-backend-module-api-docs-indexer`

Reply:
Hi @Rugvip and @backstage/techdocs-core any updates on naming conventions here?

Reply:
I think I'm good with the last proposed name from @Rugvip. I suspect it's a pattern we will want to emulate going forward, especially when we start moving search over to the new backend system.